docs/pallas/tpu/matmul.md

```diff
@@ -180,7 +180,7 @@ np.testing.assert_array_equal(x @ y, matmul(x, y))
 
 Let's think about how to analyze matrix multiplication performance.
 
-The number of FLOPs in a `(m, k) x (k, n)` matrix multiplication are (approximately) `2 * m * k * n`. (Technically it is `n * m * (2k - 1)` but for large enough `k` our approximation is sufficient.)
+The number of FLOPs (floating point operations) in a `(m, k) x (k, n)` matrix multiplication are (approximately) `2 * m * k * n`. (Technically it is `n * m * (2k - 1)` but for large enough `k` our approximation is sufficient.)
```

Comment:
What's the convention here? Is it FLOPs = total floating point operations (plural), but FLOP/s = floating point operations per second?

Maybe just call it "arithmetic operations"? Because FLOPs is also commonly used for floating point operations per second. Or explain FLOPs vs FLOP/s.